src/main/kotlin/com/ampnet/web3provider/controller/ProviderController.kt

```diff
@@ -0,0 +1,34 @@
+package com.ampnet.web3provider.controller
+
+import com.ampnet.web3provider.controller.pojo.JsonRpcRequest
+import com.ampnet.web3provider.controller.pojo.ProviderResponse
+import com.ampnet.web3provider.enums.RedisEntity
+import com.ampnet.web3provider.service.AccountInformationService
+import com.ampnet.web3provider.service.DefaultProviderService
+import mu.KLogging
+import org.springframework.http.MediaType
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+class ProviderController(
+    private val accountInformationService: AccountInformationService,
+    private val defaultProviderService: DefaultProviderService
+) {
+
+    companion object : KLogging()
+
+    @PostMapping("/provider", consumes = [MediaType.APPLICATION_JSON_VALUE])
```

Comment:
```suggestion
    @PostMapping("/api", consumes = [MediaType.APPLICATION_JSON_VALUE])
```
https://linear.app/ampnet/issue/SD-90#comment-a4cc2b26